tests/vcs/git/GitCommitBearTest.py

```diff
@@ -267,6 +269,108 @@ def test_body_checks(self):
+        self.run_git_command('remote', 'set-url', 'test',
+                             'https://github.com/user/repo.git')
+
+        # GitHub chosen and has an issue
+        self.git_commit('Shortlog\n\n'
+                        'First line, blablablablablabla.\n'
+                        'Another line, blablablablablabla.\n'
+                        'Fix https://github.com/user/repo/issues/1234')
+        self.assertEqual(self.run_uut(issue_checks=True,), [])
+
+        # Commit has neither keywords nor issues
+        self.git_commit('Shortlog\n\n'
+                        'This line is ok.\n'
+                        'This line is by far too long (in this case).\n'
+                        'This one too, blablablablablablablablabla.')
+        self.assertEqual(self.run_uut(issue_checks=True,),
+                         ['No issue related URL found in the body '
+                          'at HEAD commit. Please add one.'])
+        self.assert_no_msgs()
+
+        # GitHub chosen and no url
+        self.git_commit('Shortlog\n\n'
+                        'First line, blablablablablabla.\n'
+                        'Another line, blablablablablabla.\n'
+                        'Fix #123')
+        self.assertEqual(self.run_uut(issue_checks=True,),
+                         ['No issue related URL found in '
+                          'the body at HEAD commit. '
+                          'Please add one.'])
+        self.assert_no_msgs()
+
+        # Adding GitLab remote for testing
+        self.run_git_command('remote', 'set-url', 'test',
+                             'git@gitlab.com:user/repo.git')
+
+        # GitLab chosen and has an issue
+        self.git_commit('Shortlog\n\n'
+                        'First line, blablablablablabla.\n'
+                        'Another line, blablablablablabla.\n'
+                        'Closing https://gitlab.com/user/repo/issues/100')
+        self.assertEqual(self.run_uut(issue_checks=True,), [])
+
+        # Invalid issue number in URL
+        self.git_commit('Shortlog\n\n'
+                        'First line, blablablablablabla.\n'
+                        'Another line, blablablablablabla.\n'
+                        'Closing https://gitlab.com/user/repo/issues/1af121')
+        self.assertEqual(self.run_uut(issue_checks=True,),
+                         ['Invalid issue number present in the body '
+                          'at HEAD commit: 1af121'])
+        self.assert_no_msgs()
+
+        # Has an invalid URL
+        self.git_commit('Shortlog\n\n'
+                        'First line, blablablablablabla.\n'
+                        'Another line, blablablablablabla.\n'
+                        'Fix http://google.com/issues/hehehe')
+        self.assertEqual(self.run_uut(issue_checks=True),
+                         ['No issue related URL found in '
+                          'the body at HEAD commit. '
+                          'Please add one.'])
```

Comment:
The issue related url is present but it is not a valid issue related url... The message should be changed accordingly.